es/criteria/code-in-the-open.md

```diff
@@ -2,55 +2,55 @@
 order: 1
 ---
 
-# Código en abierto
+# Código abierto
 
 ## Requisitos
 
 * Todo el código fuente de cualquier política y software en uso (a menos que se utilice para la detección de fraudes) DEBE publicarse y ser de acceso público.
 * Los colaboradores NO DEBEN subir al repositorio información sensible sobre los usuarios, su organización o terceros.
 * Cualquier código fuente que no esté actualmente en uso (como nuevas versiones, propuestas o versiones antiguas) DEBE ser publicado.
-* El código fuente PUEDE proporcionar al público en general información sobre el código fuente o la política que sustenta cualquier interacción específica que tengan con una organización.
+* El código fuente PUEDE proporcionar al público en general información sobre qué código fuente política sustenta cualquier interacción específica que tengan con una organización.
 
 ## Por qué esto es importante
 
-Mantener el código en abierto:
+Mantener el código abierto:
 
-* mejora la transparencia
-* aumenta la calidad del código
-* facilita las auditorías
+* Mejora la transparencia.
+* Aumenta la calidad del código.
+* Facilita las auditorías.
 
 ## Qué no hace
 
 * Hace el código fuente o política reusable.
 * Hace entendibles al mayor número de personas posible la codebase y el código que contiene.
 
-## Cómo testear
+## Cómo probar
```

Comment:
I would like to keep the word "Testear". It is in the RAE dictionary (https://dle.rae.es/testear?m=form) and also refers directly to the word 'test', as in perform software checks.